internal/lsp/server.go

```diff
@@ -418,6 +419,14 @@ func (l *LanguageServer) handleInitialize(
 	}
 
 	l.clientRootURI = params.RootURI
+	l.clientIdentifier = clients.DetermineClientIdentifier(params.ClientInfo.Name)
+
+	if l.clientIdentifier == clients.IdentifierUnknown {
+		l.logError(fmt.Errorf(
```

Comment:
This doesn't seem like an error to me in a supposedly open protocol :) Can this just be a notice?

Reply:
Ahh, that should be an error. fixed in 1d516dcaacc3aecb3cf1640c555d1c11fc2c8ca9